scripts/local_testnet/validator_client.sh

```diff
@@ -12,7 +12,7 @@ DEBUG_LEVEL=${1:-info}
 exec lighthouse \
 	--debug-level $DEBUG_LEVEL \
 	vc \
-	--datadir $VALIDATORS_DIR \
-	--secrets-dir $SECRETS_DIR \
+	--datadir $DATADIR \
 	--testnet-dir $TESTNET_DIR \
 	--init-slashing-protection
```

Comment:
I think you need a `\` at the end of this line

Reply:
yeah, me too. It seemed to work without tho. I'll add it in